### ISSUE TITLE
h3SetToMultiPolygon cannot convert multipolygons correctly

### ISSUE THREAD

**Issue by boyxgc** [uber/h3-java#27]
I have a list of hex addresses:
```
89283082867ffff
8928308280fffff
8928308282bffff
8928308283bffff
89283082873ffff
89283082863ffff
89283082eafffff
89283082ea3ffff
```
which looks like this in map:
![image](https://user-images.githubusercontent.com/5845075/46251427-1e7b1c80-c407-11e8-8f78-1b574d18afb8.png)

where can see, there are two polygons in this multiPolygon, the bigger one , the smaller one. The bigger one misses the hexagon in the center, so it has a hole.

Using `h3SetToMultiPolygon` to convert the h3 above to multipolygon, the coordinates are in the form of containing only one polygon, where all 3 linear rings are in the same polygon. 

```
[
    [
        [
          coordinates ... of biggest ring
        ],
        [
          coordinates ... of hole ring
        ],
        [
          coordinates ... of island ring
        ]
    ]
]
```

While I was expecting 2 polygons in the list:
```
[
    [
        [
          coordinates ... of biggest ring
        ],
        [
          coordinates ... of hole ring
        ]
    ],
    [
        [
          coordinates ... of island ring
        ]
    ]
]
```

**Comment by isaacbrodsky** [comment]
I believe this is due to H3-Java still being on v3.1.0, the fix for this being in uber/h3#131, which is in v3.1.1. This should be fixed by #28.

**Comment by isaacbrodsky** [comment]
Hi, H3-Java v3.1.0 is now released, please reopen if this is still an issue.
